ReSharper.FSharp/src/FSharp.Common/src/Checker/FcsProjectProvider.fs

```diff
@@ -307,9 +306,8 @@ type FcsProjectProvider
 
 /// Invalidates psi caches when a non-F# project is built and FCS cached resolve results become stale
 [<SolutionComponent>]
-type OutputAssemblyChangeInvalidator
-        (lifetime: Lifetime, outputAssemblies: OutputAssemblies, daemon: IDaemon, psiFiles: IPsiFiles,
-         fcsProjectProvider: IFcsProjectProvider, scheduler: ISolutionLoadTasksScheduler) =
+type OutputAssemblyChangeInvalidator(lifetime: Lifetime, outputAssemblies: OutputAssemblies, daemon: IDaemon,
```

Comment:
Same as above.

And perhaps more to come...

Reply:
don't worry! The branch has already been rebased.